spotinst-cli

```diff
@@ -367,23 +370,19 @@ for group, group_values in groups_to_update.iteritems():
         req_path = "{}/{}/suspension".format(base_path, group_id)
 
         if options.suspend is not None:
-            if options.suspend != 'BOTH':
-                req_type = 'POST'
-                command = "suspend"
-                payload = '{"processes":["%s"]}' % options.suspend
+            req_type = 'POST'
```

Comment:
Why did you delete the !BOTH statement?

Reply:
BOTH was relevant when there was only auto scale & auto heal
now the api provides much more 
https://api.spotinst.com/spotinst-api/elastigroup/amazon-web-services/suspend-processes/
`
AUTO_SCALE, AUTO_HEALING, OUT_OF_STRATEGY, PREVENTIVE_REPLACEMENT, REVERT_PREFERRED
`

Reply:
I would leave BOTH for now since this is our most used use-case.

Reply:
so... it's still deleted?

Reply:
nope, its back :) 
because it is our most important usage